Assets/MixedRealityToolkit.Services/InputSimulation/SimulatedHandPose.cs

```diff
@@ -158,6 +158,10 @@ public enum GestureId
             /// Victory sign
             /// </summary>
             Victory,
+            /// <summary>
+            /// Open Sky
```

Comment:
Better description needed, e.g. "Open hand with palm facing up and fingers spread out".

Reply:
Absolutly, "Open Sky" means nothing